backend/database/seeding/test_helpers.go

```diff
@@ -228,22 +230,99 @@ func GetOperationFromSlug(t *testing.T, db *database.Connection, slug string) mo
 	return fullOp
 }
 
-func GetOperations(t *testing.T, db *database.Connection) []models.Operation {
-	var fullOps []models.Operation
-	err := db.Select(&fullOps, sq.Select("id", "slug", "name", "status").
-		From("operations"))
+func GetOperations(t *testing.T, db *database.Connection) []services.OperationWithID {
+	var operations []struct {
+		models.Operation
+		NumUsers    int `db:"num_users"`
+		NumEvidence int `db:"num_evidence"`
+		NumTags     int `db:"num_tags"`
+	}
+
+	var topContribs []dtos.TopContrib
+
+	var evidenceCount []dtos.EvidenceCount
+
+	err := db.Select(&operations, sq.Select("operations.id", "slug", "operations.name", "status", "count(distinct(user_operation_permissions.user_id)) AS num_users", "count(distinct(evidence.id)) AS num_evidence", "count(distinct(tags.id)) AS num_tags").
+		From("operations").
+		LeftJoin("user_operation_permissions ON user_operation_permissions.operation_id = operations.id").
+		LeftJoin("evidence ON evidence.operation_id = operations.id").
+		LeftJoin("tags ON tags.operation_id = operations.id").
+		GroupBy("operations.id").
+		OrderBy("operations.created_at DESC"))
+
 	require.NoError(t, err)
-	return fullOps
+
+	err = db.SelectRaw(&topContribs, services.GetTopContributorsForEachOperation)
+
+	require.NoError(t, err)
+
+	err = db.SelectRaw(&evidenceCount, services.EvidenceCountForAllOperations)
+
+	require.NoError(t, err)
+
+	operationsWithID := []services.OperationWithID{}
+	for _, operation := range operations {
+
+		var topContribsForOp []dtos.TopContrib
+		for i := range topContribs {
+			if topContribs[i].OperationID == operation.ID {
+				topContribsForOp = append(topContribsForOp, topContribs[i])
+			}
+		}
+
+		var evidenceCountForOp dtos.EvidenceCount
+		for i := range evidenceCount {
+			if evidenceCount[i].OperationID == operation.ID {
+				evidenceCountForOp = evidenceCount[i]
+				break
```

Comment:
You might be able to use Find for this. Something like:

```go
evidenceCountForOp := helpers.Find(evidenceCount, func(ec dtos.EvidenceCount){ return ec.OperationID == operation.ID } )
```

Reply:
hmmm...had some type issues come up that I was able to fix, but I'm glad to know these helpers exist